distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Date_Time_Helpers.enso

```diff
@@ -9,30 +9,31 @@ import project.Column.Column
 import project.Value_Type.Value_Type
 
 ## PRIVATE
-check_period_aligned_with_value_type value_type period ~action = case value_type of
+align_period_with_value_type value_type period = case value_type of
```

Comment:
```suggestion
align_period_with_value_type value_type period:(Date_Period|Time_Period) = case value_type of
```

Reply:
It's an internal method so I thought there is no need to do this - the autoscopes should already have been resolved by the public API.

Reply:
But can add.